chart/openfaas/templates/gateway-dep.yaml

```diff
@@ -28,8 +28,17 @@ spec:
           initialDelaySeconds: 2
           periodSeconds: 10
           timeoutSeconds: 2
-
+        {{- if .Values.gateway.basicAuth }}
+        volumeMounts:
+        - name: basic-auth
+          mountPath: "/secrets"
+          readOnly: true
+        {{- end }}
         env:
+        - name: basic_auth
+          value: "{{ .Values.gateway.basicAuth }}"
+        - name: secret_mount_path
+          value: /secrets
```

Comment:
Can you make this `/var/run/openfaas/secrets` to match what we use for functions?

Reply:
Yes

Reply:
Fixed in second commit. :)